hpx/runtime/actions/manage_object_action.hpp

```diff
@@ -9,17 +9,18 @@
 
 #include <cstring>
 #include <boost/config.hpp>
-#include <boost/serialization/serialization.hpp>
 
 #include <hpx/config/warnings_prefix.hpp>
 #include <hpx/exception.hpp>
-#include <hpx/util/portable_binary_oarchive.hpp>
-#include <hpx/util/portable_binary_iarchive.hpp>
 #include <hpx/runtime/actions/action_support.hpp>
-#include <hpx/util/void_cast.hpp>
-#include <hpx/util/base_object.hpp>
+#include <hpx/runtime/serialization/array.hpp>
+#include <hpx/runtime/serialization/base_object.hpp>
+#include <hpx/runtime/serialization/serialize_buffer.hpp>
 #include <hpx/util/reinitializable_static.hpp>
 
+#define HPX_SINGLE_ARG(...) __VA_ARGS__
+#define HPX_COMMA ,
```

Comment:
This doesn't seem to be the proper place for these macro definitions, nor they seem to be used anywhere either. Are they still needed?